### DISCUSSION TITLE
LTBEEF Alternatives?

### DISCUSSION THREAD

**Discussion by 6829ASCS** [3kh0/ext-remover#109] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
After avoiding the new update for days, my Chromebook decided to run out of charge and turn off. 

Now that LTBEEF doesn't work for me anymore, **are there any alternatives to achieve the same result and disable admin extensions**? Please don't suggest using proxy sites or HU links because I've tried those and they're super inconvenient. 

**_Side note:_** 

I recently experimented with this bookmark javascript code thing that creates a hidden page in which you can search and access websites using bing without it showing up on your history. This proved to be extremely useful except pages on it still get blocked. Is there any way this could be avoided since this seems to be the optimal solution right now?

![image](https://user-images.githubusercontent.com/88504752/197331132-0cfdfaa8-a185-40e9-b628-3a04b5bc769e.png)

**Comment by 6829ASCS** [comment]
Quick update, the alternative I mentioned is a web proxy that remains completely hidden from your history but can still get blocked. Somewhat useful I guess if you don't want stuff popping up on your search history? 

If anyone is interested I can pop the javascript for it here..

**Reply by JelloFruitAlt** [reply]
yeah, i would be interested please

**Reply by Fallenmoon8080** [reply]
May I see this javascript please.

**Reply by Fallenmoon8080** [reply]
Here's his history hider, which can be found here: https://sites.google.com/site/hubeway/inside-a-tab-tab

**Reply by JelloFruitAlt** [reply]
doesn't work, im not sure if my filter blocks it tho.

**Reply by 6829ASCS** [reply]
Here, this is a bookmarklet that creates a hidden Bing page that doesn't show up on your history. Things still get blocked on there but I guess its useful when paired with other stuff or when you want to keep stuff you search hidden, especially at school.

I experimented with it quite a bit and some useful scenarios for it would be using those unblocked game websites through the hidden browser, that way it doesn't show up on your history. 

`javascript:let urlQueue=[],backButton=document.createElement("button");backButton.style.width="25px",backButton.innerHTML="<";let urlBox=document.createElement("input");urlBox.type="text",urlBox.style.width="480px";let goButton1=document.createElement("button");goButton1.innerHTML="Load Page by iFrame";let goButton2=document.createElement("button");goButton2.innerHTML="Load Page by Proxy";let ytButton=document.createElement("button");ytButton.innerHTML="Load YouTube";let iframe=document.createElement("iframe");iframe.src="https://www.bing.com",iframe.width=window.innerWidth,iframe.height=window.innerHeight,goButton1.addEventListener("click",(()=>{0!=urlBox.value.length&&urlBox.value.startsWith("http")&&(iframe.src=urlBox.value.toLowerCase(),urlQueue.push(urlBox.value.toLowerCase()))})),goButton2.addEventListener("click",(()=>{iframe.src="https://www.proxysite.com",urlQueue.push("https://www.proxysite.com")})),ytButton.addEventListener("click",(()=>{iframe.src="https://piped.kavin.rocks/",urlQueue.push("https://piped.kavin.rocks/")})),backButton.addEventListener("click",(()=>{urlQueue.length>1&&(urlQueue.pop(),iframe.src=urlQueue.at(-1))})),document.body.appendChild(backButton),document.body.appendChild(urlBox),document.body.appendChild(goButton1),document.body.appendChild(goButton2),document.body.appendChild(ytButton),document.body.appendChild(iframe);`

**Reply by JelloFruitAlt** [reply]
thank you very much :>

**Comment by KyraCoding** [comment]
Yeah me too

**Comment by ghost** [comment, marked as answer]
not any that actually remove an extension, at least that I know of.
